### PR TITLE
Feature/cce 453

### PR DESCRIPTION
**Pull request by HybrisRawls**

### CODE REVIEW

File: src/less/overwrites/__overwrites.less [hunk 1]
```diff
@@ -36,6 +36,15 @@ h6{
    }
 }
 
+h1{
+    font-weight: 400 !important;
```
Comment (taifunhaeberle) [inline src/less/overwrites/__overwrites.less]: Please take a look at https://github.com/twbs/bootstrap/blob/master/less/type.less we should also add the .h1 there. Also i doubt we need the important? ;)

File: src/less/overwrites/__overwrites.less [hunk 2]
```diff
@@ -36,6 +36,15 @@ h6{
...
+}
+
+
+h2{
+    font-weight: 600 !important;
```
Comment (taifunhaeberle) [inline src/less/overwrites/__overwrites.less]: Please take a look at https://github.com/twbs/bootstrap/blob/master/less/type.less we should also add the .h2 there. Also i doubt we need the important? ;)

File: src/less/overwrites/_modal.less [hunk 3]
```diff
@@ -8,7 +8,7 @@
 //     <div class="modal-content">
 //       <div class="modal-header">
 //         <button type="button" class="close" data-dismiss="modal">&times;</button>
-//         <h4 class="modal-title">Modal Header</h4>
+//         <h1 class="modal-title">Modal Header</h1>
```
Comment (taifunhaeberle) [inline src/less/overwrites/_modal.less]: I'm not sure if this is a good idea, as the modal is part of the normal dom tree of a page. This means we end up with two <h1>'s even if the modal is not visible. Maybe here a special style for the modal header would be more fitting?

File: docs/template/public/less/_variables.less [hunk 4]
```diff
@@ -22,4 +22,11 @@
 @dark-blue: #27394f;
 @side-nav-link: #dbe3eb;
 @top-nav-hover: #0376C6;
-@btn-type-color: #0b0080;
\ No newline at end of file
+@btn-type-color: #0b0080;
+@kss-green: #04ce26;
```
Comment (taifunhaeberle) [inline docs/template/public/less/_variables.less]: We really should separate kss (techne page) variables form the actuall framework

File: docs/template/public/kss.less [hunk 5]
```diff
@@ -1,4 +1,4 @@
-@import url(https://fonts.googleapis.com/css?family=Dosis:400,300,600,700);
+@import url(https://fonts.googleapis.com/css?family=Dosis:400,500,600);
```
Comment (taifunhaeberle) [inline docs/template/public/kss.less]: I doubt we need to include that at all here, as we have the techne from distribution folder included?

File: docs/template/public/less/_variables.less (hunk 4 above)
Comment (HybrisRawls) [inline docs/template/public/less/_variables.less]: good catch, this should be in the css variables

File: src/less/overwrites/__overwrites.less (hunk 1 above)
Comment (HybrisRawls) [inline src/less/overwrites/__overwrites.less]: done.

File: src/less/overwrites/__overwrites.less (hunk 2 above)
Comment (HybrisRawls) [inline src/less/overwrites/__overwrites.less]: done

File: src/less/overwrites/_modal.less (hunk 3 above)
Comment (HybrisRawls) [inline src/less/overwrites/_modal.less]: should we use a h2? The h4 doesn't really make sense to me

File: docs/template/public/kss.less (hunk 5 above)
Comment (HybrisRawls) [inline docs/template/public/kss.less]: agreed

File: docs/template/public/less/_variables.less (hunk 4 above)
Comment (taifunhaeberle) [inline docs/template/public/less/_variables.less]: @saadmhybris Please move variables that are used exclusively for the page styling out of the normal variables file. into the kss.less. If you want to use pre-existing variables of the framework in the kss.less please make a import of the less file.

Comment (saadmhybris) [inline docs/template/public/less/_variables.less]: @HybrisRawls @taifunhaeberle  - isn't this file only for kss site? I talked to jeremy about this and he said not to import frame work variables here to keep things simple.

Comment (taifunhaeberle) [inline docs/template/public/less/_variables.less]: @saadmhybris that's right. My bad !

**Comment by HybrisRawls** [comment]
@taifunhaeberle  is this one ready to merge?

File: src/less/overwrites/_type.less [hunk 6]
```diff
@@ -1,5 +1,22 @@
+
+h1{
```
Comment (taifunhaeberle) [inline src/less/overwrites/_type.less]: You are lacking the .h1 classes from bootstrap ( and all following). Please have a look at the typefile of bootstrap

Comment (saadmhybris) [inline src/less/overwrites/_type.less]: done. @HybrisRawls this is ready for merge now i believe.

**Pull request closed without merging**